purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -76,13 +76,13 @@ import org.junit.runner.RunWith
 import org.robolectric.annotation.Config
 import java.net.URL
 import java.util.Collections.emptyList
-import java.util.ConcurrentModificationException
 import java.util.concurrent.CountDownLatch
 import java.util.concurrent.TimeUnit
 import kotlin.random.Random
 
 @RunWith(AndroidJUnit4::class)
 @Config(manifest = Config.NONE)
+@Suppress("DEPRECATION")
```

Comment:
I added this since I was getting confused with a lot of the deprecated warnings we have (since we need to be testing the deprecated code as well)